spoon-runner/src/main/java/com/squareup/spoon/SpoonRunner.java

```diff
@@ -99,6 +101,91 @@ private SpoonRunner(String title, File androidSdk, File applicationApk, File ins
     }
   }
 
+  /** Returns {@code false} if a test failed on any device. */
+  static boolean parseOverallSuccess(SpoonSummary summary) {
```

Comment:
looks like you relocated all the static methods...can you move them back where they were so the diff is easier to grok?